otter/integration/config.sh-template

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# To run the integration tests, you'll need to have various environment
+# settings established.
+
+# This is the username used to authenticate against Identity.
+export AS_USERNAME=joe_user
+
+# This is the password used to authenticate against Identity.
+export AS_PASSWORD=joes_password
```

Comment:
I'm sorry to block this on something, but it's not a mere suggestion; given the likelihood of credentials leaks, I'm not comfortable letting this land without some good motivation as to why this is unlikely to result in leaked credentials. "There's a pre-commit hook for that" is good motivation :)